doc/api/cli.md

```diff
@@ -216,6 +216,18 @@ reference. Code may break under this flag.
 added: v12.0.0
 -->
 
+Enables a signal handler that causes the node process to write a heap dump when the specified signal is received.
```

Comment:
Pretty sure the markdown linter will fail on this line because it is longer than 80 characters. Can you wrap it?

You can run the markdown linter locally with `make lint-md` (or, on Windows, `vcbuild lint-md`).